common/src/test/java/org/apache/celeborn/common/network/RpcIntegrationSuiteJ.java

```diff
@@ -101,6 +107,7 @@ public boolean checkRegistered() {
   public static void tearDown() {
     server.close();
     clientFactory.close();
+    context.close();
     testData.cleanup();
   }
```

Comment:
The existing test suites, like `RpcIntegrationSuite`, `TransportClientFactorySuite`, etc are modified such that a SSL subclass can test all the cases - but over SSL.